src/sass/app/components/peek_box.scss

```diff
@@ -0,0 +1,306 @@
+        }
+    }
+    .peek-box.digit-losing {
+        @each $item in $PEEKBOX_GRID_ITEMS {
+            .#{$item} {
+                background-color: $COLOR_PEEK_RED;
+            }
+        }
+
+        .peek {
+            border: 5px solid $COLOR_PEEK_RED;
+        }
+        .mask {
+            border: 1px solid $COLOR_PEEK_RED;
+        }
+    }
+    .peek-box.digit-winning {
+        @each $item in $PEEKBOX_GRID_ITEMS {
+            .#{$item} {
+                background-color: $COLOR_PEEK_GREEN;
+            }
+        }
+
+        .peek {
+            border: 5px solid $COLOR_PEEK_GREEN;
+        }
+        .mask {
+            border: 1px solid $COLOR_PEEK_GREEN;
+        }
+    }
+    .digits {
+        display: flex;
+        align-items: center;
+        min-width: 360px;
+        justify-content: space-evenly;
+        box-sizing: border-box;
+        margin: 0 0 3px 0px;
+        border-radius: 4px;
+        overflow: hidden;
+
+        @media screen and (max-width: 480px) {
+            min-width: 280px;
+        }
+    }
+    .digit.digit-winning {
+        background-color: $COLOR_DIGIT_WINNING_BACKGROUND;
+        color: $COLOR_DIGIT_WINNING;
+        border: none;
+    }
+    .digit.digit-lost {
+        color: $COLOR_PEEK_RED;
+    }
+    .digit.digit-won {
+        color: $COLOR_PEEK_GREEN;
+    }
+    .digit {
+        height: 36px;
+        width: 36px;
+        font-size: 20px;
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        font-weight: bold;
+        border-right: 1px solid $COLOR_GRAY;
+        border-top: 1px solid $COLOR_GRAY;
+        border-bottom: 1px solid $COLOR_GRAY;
+        color: $COLOR_PEEK_LIGHT_GRAY;
+
+        @media screen and (max-width: 480px) {
+            height: 28px;
+            width: 28px;
+        }
+
+        &:first-child {
+            border-left: 1px solid $COLOR_GRAY;
+            border-top-left-radius: 4px;
+            border-bottom-left-radius: 4px;
+        }
+        &:last-child {
+            border-top-right-radius: 4px;
+            border-bottom-right-radius: 4px;
+        }
+    }
+}
+
+.row.digit-trade {
+    & .col {
+        font-size: 42px;
+        color: $COLOR_GRAY;
+        position: relative;
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+
+        & .quote {
+            width: auto;
+            color: $COLOR_PEEK_LIGHT_GRAY;
+            font-weight: 100;
+            margin-top: 2px;
+        }
+        & .digit-tick-epoch {
+            display: inline-block;
+            vertical-align: top;
+            margin-right: 2px;
+            top: 14px;
+            font-size: 14px;
+            line-height: 14px;
+            color: $COLOR_PEEK_RUNNING_GRAY;
+            padding-left: 14px;
+            padding-right: 2px;
```

Comment:
you can remove `padding-right: 2px` as well, the point was to right align timestamp with the `.quote`

Reply:
all done.